purchase_order_secondary_unit/views/purchase_order_views.xml

```diff
@@ -6,26 +6,27 @@
         <field name="name">Purchase Order Secondary Unit</field>
         <field name="model">purchase.order</field>
         <field name="inherit_id" ref="purchase.purchase_order_form" />
-        <field name="groups_id" eval="[(4, ref('uom.group_uom'))]" />
         <field name="arch" type="xml">
             <xpath
                 expr="//field[@name='order_line']/form//field[@name='product_qty']"
                 position="before"
             >
-                <field
-                    name="secondary_uom_qty"
-                    class="oe_inline oe_no_button"
-                    attrs="{'readonly': [('state', 'in', ('done', 'cancel'))]}"
-                />
-                <field
-                    name="secondary_uom_id"
-                    class="oe_inline"
-                    domain="['|', ('product_id', '=', product_id),
-                                '&amp;', ('product_tmpl_id.product_variant_ids', 'in', [product_id]),
-                                         ('product_id', '=', False)]"
-                    options="{'no_create': True}"
-                    attrs="{'readonly': [('state', 'in', ('purchase', 'done', 'cancel'))], 'required': [('secondary_uom_qty', '!=', 0.0)]}"
-                />
+                <t groups="uom.group_uom">
```

Comment:
Why isn't the same `t` tag needed in the other `xpath` below?

Reply:
Added group coverage for missing fields (`groups` attribute instead of `<t>`, that cannot be used under a `<tree>` view)

Reply:
Thanks